core/sql/sqlmxevents/logmxevent_traf.h

```diff
@@ -189,4 +190,7 @@ class SQLMXLoggingArea
 																									 
 };
 
+void logAnMXEventForError( ComCondition & condition, SQLMXLoggingArea::ExperienceLevel emsEventEL);
```

Comment:
Is there a function body somewhere that goes with this? I don't see it in this pull request.

Reply:
The implementation was already available at cli/CliExtern.cpp. The corresponding include file will be sqlcli.h that contains the interface to be used by the clients to Trafodion SQL engine. Hence, I added this declaration in logmxevent_traf.h